src/layerData.json

```diff
@@ -7,15 +7,7 @@
     },
     "description": "The World Air Quality Index project provides transparent and world-wide air quality information for more than 100 countries, covering more than 12,000 stations in 1000 major cities.",
     "layer_desc": "World-wide air quality information",
-    "icon": "#096",
-    "extents": {
-      "bounds": [
-                  [-44.087585028245165, -148.88671875000003],
-                  [76.63922560965888, 140.62500000000003]
-                ],
-      "minZoom": 3,
-      "maxZoom": 15
-    }
+    "icon": "#096"
```

Comment:
OK great! Did you manually check these to ensure they should work outside the previously created bounds? Forgive me, i forget - did we figure out how these bounds were originally established? They're huge!

Reply:
I got the bounds for the initial map view and added it to the json file for those layers that didn't have bounds. And yes, I manually checked every layer added to the demo page and it works.

Reply:
It didn't occur to me that I could code it to work when there were no bounds given at the time I wrote it. 😅

Reply:
I got the bounds for JusticeMap layer by running `map.getBounds()`. Bounds and zoom for other layers were taken from the info.json file. I am not sure how they were established.

Reply:
Yes they were figured the same way you did 😄